### DISCUSSION TITLE
why collect information without informing users

### DISCUSSION THREAD

**Discussion by WindSmileValley** [crewAIInc/crewAI#1094] [Q&A]
why collect information without informing users ? such as, telemetry.  I am not sure if there are other issues as well.
 i suggest that by default , no information should be collected unless actively allowed to do so. this feature also reflects a lack of respect for personal privacy from the author.

**Comment by zinyando** [comment]
> why collect information without informing users ? such as, telemetry. I am not sure if there are other issues as well. i suggest that by default , no information should be collected unless actively allowed to do so. this feature also reflects a lack of respect for personal privacy from the author.

They do not collect any private data, and it's anonymous. I guess they want to be able to track usage data that will allow them to refine the product appropriately. A few people have complained about this before and they have addressed it.

I think adding the ability to turn off telemetry would be great and it would smoothen up enterprise adoption. You can read more about telemetry here https://docs.crewai.com/telemetry/Telemetry/

**Comment by WindSmileValley** [comment]
> > why collect information without informing users ? such as, telemetry. I am not sure if there are other issues as well. i suggest that by default , no information should be collected unless actively allowed to do so. this feature also reflects a lack of respect for personal privacy from the author.
> 
> They do not collect any private data, and it's anonymous. I guess they want to be able to track usage data that will allow them to refine the product appropriately. A few people have complained about this before and they have addressed it.
> 
> I think adding the ability to turn off telemetry would be great and it would smoothen up enterprise adoption. You can read more about telemetry here https://docs.crewai.com/telemetry/Telemetry/

I fully endorse your recommendation. and this feature is still exit.

**Comment by theCyberTech** [comment]
Users are informed - https://docs.crewai.com/telemetry/Telemetry/.

Also this is an open source product all the code is public.

Can you please expand on what  personal privacy is being impeded?

**Reply by WindSmileValley** [reply]
You should understand that not refusing does not mean agreeing, let alone asking explicitly.

**Reply by theCyberTech** [reply]
not sure what that even means, as mentioned previously it is all explained in the docs - https://docs.crewai.com/telemetry/Telemetry/

So if you could please elaborate on what personal information is impeded, that would be appreciated
